subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -147,21 +161,46 @@
    * </pre>
    *
    * This method attempt to schedule the calling task to current plan's scheduler
-   *    and it can be executed asynchronously
+   *    and it can be executed asynchronously.
    *
    * If the method was called when no plans are running,  the calling cannot be scheduled
    *   to any plans, and it will throws Exception in this case.
    *
+   * Note the task calling {@link#scheduleAndRun} would be running as a sibling task of the
```

Comment:
Forgot to update comment?

Reply:
Hi Min I think I have updated this (the version you are checking is an outdated one)

Reply:
Example code is not updated